intersection_transit_maneuvering/src/call_test.cpp

```diff
@@ -0,0 +1,29 @@
+#include "call_test.h"
+#include <ros/ros.h>
+
+namespace call_test
+{
+
+    CallTest::CallTest(){};
```

Comment:
When you have an empty constructor you can actually just have CallTest() = default; in the header file.